chore: fix cspell missing words

owlot opened the pull request:
I noticed the cspell check was failing on some valid words, so I added them to the .cspell.project-words.txt file.
In addition, the word `trackability` was being used where I think `tracability` might be a better word, but that could be wrong, so let me know if I need to change that back before merging this PR.

owlot committed:
chore: fix cspell missing words

sl33ty commented:
> I noticed the cspell check was failing on some valid words, so I added them to the .cspell.project-words.txt file. In addition, the word `trackability` was being used where I think `tracability` might be a better word, but that could be wrong, so let me know if I need to change that back before merging this PR.

The only legitimate spelling I've found is `traceability`, with a 'e' (https://en.wikipedia.org/wiki/Traceability)

owlot commented:
> > I noticed the cspell check was failing on some valid words, so I added them to the .cspell.project-words.txt file. In addition, the word `trackability` was being used where I think `tracability` might be a better word, but that could be wrong, so let me know if I need to change that back before merging this PR.
> 
> The only legitimate spelling I've found is `traceability`, with a 'e' (https://en.wikipedia.org/wiki/Traceability)

Fixed!
